docs/monitoring/README.md

```diff
@@ -2,13 +2,4 @@
 
 After deploying a software solution, the task becomes monitoring it. With the nio Platform, it is easy to monitor the environment of nio instances. But in many cases, this is not granular enough for your needs because there is no visibility of the environment at the nio-service level.
```

Comment:
I know this was already there, but I think that first sentence is a little confusing because we say `but with the nio Platform it's easy to monitor the environment of nio instances` but it's actually also easy to monitor nio services with the monitoring tool now. I think we should just reword this a little!